wordCreator.py

```diff
@@ -0,0 +1,294 @@
+import numpy as np
+import random
+
+class WordCreator(object):
+    def __init__(self, filename=None, words=None):
+        """
+        Initializes variables in WordCreator object
+        Creates Transition matrix and normalizes it
+        Takes string path (filename), or string (words)
+        """
+
+        #Check whether input was text file or list of words
+        if filename:
+            self.txtFile = open(filename, 'r')
+            self.words = None
+        else:
+            self.txtFile = None
+            self.words = words
+
+        #Initialize attributes
+        self.transitions = np.zeros([256, 256])
+        self.sumList = []
+        self.minLetters = self.getMinLetters()
+        self.maxLetters = self.getMaxLetters()
+
+        #Run through training set and assign probabilities
+        self.setTransitions()
+
+    def getMaxLetters(self):
+        """
+        Returns the max number of letters in a word in the text file
+        """
+
+        currentMax = 0
+
+        #If reading from text file
+        if self.txtFile:
+            self.txtFile.seek(0) #Go to beginning of file
+            #Loop through file
+            for word in self.txtFile:
+                #Find longest word
+                if len(word) >= currentMax:
+                    currentMax = len(word)
+        else:
+            currentMax = 15
+        #Else read from word list
+        # else:
+        #     #Loop through list
+        #     for word in self.words:
+        #         #Find longest word
+        #         if len(word) >= currentMax:
+        #             currentMax = len(word)
+        # #Return length of longest word
```

Comment:
It is a good practice to delete the unncecssary, commented out code that is not a part of your final program.